depends: Update URL for `qrencode` package source tarball

hebasto opened the pull request:
The https://fukuchi.org/ homepage no longer links to the source tarball, and previously available files appear to have been removed. The homepage now instructs users to download source tarballs from the GitHub [releases](https://github.com/fukuchi/libqrencode/releases) page instead.

The diff between the source trees is immaterial:
```diff
--- old
+++ new
@@ -1,19 +1,16 @@
 27e7deccd2925c94e4190ee64794a051199f215f145f76fd664cdebedbbf8a35  acinclude.m4
-e1e35b1309482f699a9700a2065a0bce09c2108dd1f78ba7bfbe0f7f0bdcd2e6  aclocal.m4
 a9308eec78790720dbcd5452ab8f241b5f1c6939ccf3389917b8e78cb2b58c9e  autogen.sh
 aa36725d577048f0370dc7415a1acb578fbdfb531c1b384a836d9360a81f6f5c  bitstream.c
 3feaacd7d096834fc5956215598564ec287ad443185c4433c3f8007cc53ceaa6  bitstream.h
 21bd5a34c90d3d6ee540ceb48c3d5aea5f21bd6b829ef3112db832af0bd423d8  ChangeLog
 f8a7a94c9622fab721df47e8121533ebfcb79885aca01ecec2fff00dfb84caef  cmake/FindIconv.cmake
 6345e7eecb92473f361a8eb98dd373aa09cae79a43408cf4b42b00b411c9c197  CMakeLists.txt
-031d560570eab2eda57c2f9ee9952445002f8bf4a23965fcaf43bbc2c61590e2  config.h.in
-d2f42cc5771b69f2d2ecd4b31509864ae2b18b25c823986390bb372ee07030fa  configure
 02867a8ea08206c84d5f4c05f41a15c639291091e3dfac27f3a9029d8f5d3028  configure.ac
 a9bdde5616ecdd1e980b44f360600ee8783b1f99b8cc83a2beb163a0a390e861  COPYING
 76c41754bccbf69a60fb7833776637c60b86f59104705c0a5cea9ee3a2968f3d  Doxyfile
 36d84f714cf28397b02d6c44860106a7cb858fc6d25239a2698a72fa5136e5c3  libqrencode.pc.in
 f0ce93a7e1b1f0fab87dd071ec3c8dd80a567d778dfc5930d9f375d676ecb9a0  Makefile.am
-507c2385fb49c5724e8e3dfca97feb24cbdd4e651f3f0bfa62524e60088091a0  Makefile.in
+dcb782b1b382328c0bce9194944bdcc65a6035d8e9f89e39436ed2e55ae8b969  makeREADME.sh
 6046d347c6c564fc13a24dc0a15a09a83023e00a4e0d0f23029a81f86ac4d024  mask.c
 9f853cee7d72191a8dbf018b7a86c0f0b6ca661e27a51fab677af911f2ff9e7e  mask.h
 a4e17b68d8db573e152132ebcbdc837b55415c12027f81232662db645faf79f0  mmask.c
@@ -30,7 +27,7 @@
 6e9ff66002b4a839c6e78bcd6a55342d9eb8b289273fa838441b27ee9969e293  qrinput.h
 68831e02ee1ba602b1937328abec000e616c4472d9dc40067dd45ab7072df172  qrspec.c
 a2a9a5af4d62015e82b48b8316aea2b70031dcdc1ed2b829c0102ea5dc02aca2  qrspec.h
-8730d006f1d45b90dc0ef9a20e4119420f15b37a09483c929baa4225e21f7900  README
+a56773b55989ea5cae8a43f2d845ce0afa9576a26170937fbe36a69fee953cd4  README.md
 ace480f2e16001e276c73a5e965c282915ad81e2c28cce3ef574b5a8db7210e8  rsecc.c
 c731ebe26d58d5e5df3b2d694de0fb1c4ae80a36cf0559c49b04e989dee8b182  rsecc.h
 8ed1af4414b5628845519581c82bf51ba2b4dee1f8352c1fd01b50afcd0a0ed7  split.c
@@ -46,7 +43,6 @@
 e26b20198a7393b3c060891876fa45edb81488aef9df6d6cf45893a605e5e5e4  tests/decoder.h
 adabfbddb0f25de2e2cae970195fcfdf11ad1fa66ce64e237d83f821346f64c4  tests/frame
 23c283a3d3b5f1bbb3108603ebf05d7a95951ba86a288e09140eab612a5eb258  tests/Makefile.am
-c8b819bf7f00c979ab0b6f0d0fb10289d2006bc6acda1737d2f1c383a60e1e66  tests/Makefile.in
 ebd71b937d4d39f382f8c57f362a01f11fd617b66dbdf6f1a3c4897c8c42e235  tests/prof_qrencode.c
 d4adcb234d4c31473cd9ffedd1bfa9645ba336d426640ba5893dfe6e75db171d  tests/pthread_qrencode.c
 62298641504b0a2c1d199cd7f656cb376959200169042de76f7c5950d2d42395  tests/rscode.c
@@ -63,6 +59,7 @@
 cf5792cbfb92ba46f7f5f14b98f00813b0e7e4f5e7c790c6a28793774474bf26  tests/test_monkey.c
 3a2c58346d57f6bb2a634d6febf3c65ac524259a2d789657f8d60c678bf8d658  tests/test_mqrspec.c
 b9d8c569ba36a2258cb1ad0d1f85ce4fb2935fad519e902bade59aa9772321aa  tests/test_qrencode.c
+785fe14a6f8bc096b20ec271771fb09f22f29eb4f9d729f5b40d6b4cc824ce36  tests/test_qrenc.sh
 d0eee6eddf98ee4595c07f7da40c7de548651bf839b26995756e94db2599451f  tests/test_qrinput.c
 ace7885e435ef77cc127da0ac23c724498aaa8d80d53b908063f79c9f0b7acce  tests/test_qrspec.c
 fde9c2735ce94be51e1b0bbffe65415aa3afd2da5bd4c0e2c5e50e1a2f1ea3f7  tests/test_rs.c
@@ -71,12 +68,4 @@
 6f35c0e1235b31d0068c4ab175d8110c736e60df0309d4be7b3e57dd62d316f6  tests/URI_testset.inc
 6be3983fc397cd5dade1dd219ad6cbe7977f416410b1509984006ecec51605b5  tests/view_qrcode.c
 92b5be1ca2239399232d51503715c848dae9bf3db71b1f03157bfa9779826910  TODO
-c8af04e62bad4ab75dafd22119026e5e3943f385bdcbe7731a4938102453754c  use/compile
-1d9048b0ac9d4d1dfce7aa4e3a0b59ccfd32db2f7693814aa6f4c778560f5669  use/config.guess
 b522487f9c47661d321367d133f3d41247dd16d435f2d4b9c643dee95bf65eee  use/config.rpath
-5bf0da2576ebb21ab60a9d9291a85b40af0e956a9eafb709ca8b20dcb105f4ee  use/config.sub
-732bcd6b9e23f241e015d71b0a3a862104053aba20718c1f56b292cee7e29371  use/depcomp
-608b76d735bb2ec2bcb1271644c3d5e7a428fb8d2338e114e8a48ebf91ccfd23  use/install-sh
-2304d53af1f63b76a11651efdd18578adf2bda1ffc2c257100cba374b55f284b  use/ltmain.sh
-f038345dab184e538098d22a8edc423762da66a90ebe269f23bfef85287cd30c  use/missing
-7c1ae35455771ae32050c2ed109e3d297160b6d1a2f70b0278cf6968e5e7e98c  use/test-driver
```

---

**UPDATE 2025-09-30**

A few commits have been added:

1. The first commit fixes the `$(package)_fetched` target, which erroneously succeeds on the second run after a failure on the master branch:
```
$ gmake -C depends clean-all
$ gmake -C depends qrencode_fetched FALLBACK_DOWNLOAD_PATH="https://fallback.invalid"
$ gmake -C depends qrencode_fetched FALLBACK_DOWNLOAD_PATH="https://fallback.invalid"
$ echo $?
0
```
With the first commit applied:
```
$ gmake -C depends clean-all
$ gmake -C depends qrencode_fetched FALLBACK_DOWNLOAD_PATH="https://fallback.invalid"
$ gmake -C depends qrencode_fetched FALLBACK_DOWNLOAD_PATH="https://fallback.invalid"
$ echo $?
2
```

2. The second and third commits allow the depends build subsystem to detect when the source tarball content has been modified and needs to be re-downloaded, even if the file name remains the same.

hebasto commented:
> I believe it's a cache issue.

Resolved by using the new `$(package)_file_name`.

m3dwards commented:
> Can the `depends` CI cache be invalidated: (1) for this PR; (2) globally?

I would have thought the cache should have been invalidated automatically, looking into it.

vasild reviewed (approved):
ACK 980e4987db0450cabb8761433001841f7e9f4c73

I confirm the diff from the OP. The old archive is not available anymore at https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.gz, so I downloaded it from https://bitcoincore.org/depends-sources/qrencode-4.1.1.tar.gz instead. The new one is at https://github.com/fukuchi/libqrencode/archive/refs/tags/v4.1.1.tar.gz.

Then I extracted each archive, did
```sh
for f in $(find ./ -type f |sort) ; do sha256sum "$f" ; done
```
and compared the resulting diff to the OP. It is the same, except some more dot-files in the new archive:
```diff
+a11819fcd7107f6b45336207762a362dbd83ce46c06b5c126d38c98860b0f5a3 ./.github/workflows/cmake-windows.yml
+e8e213ffe4d032685b3215feb6c1cb821442f0bc5a2d00f147a4f474bb5de2d2 ./.github/workflows/cmake.yml
+8db8a290dbcb99d04ec021f107e50c0da7f84f461b251955869f7d69174b54e4 ./.github/workflows/configure.yml
+a1f246edb6aedbc908987bd04e0743f4ebf415858b10bf81b6a7969dd6bf2409 ./.gitignore
+4eff4a00c6a9f09c9fb39751dc6666b8a6d845de0a4fc6b38522740b6a262e41 ./.travis.yml
```

hebasto committed:
depends: Fix `$(package)_fetched` target

Ensure the download timestamp is created only after a successful
download.

hebasto committed:
depends: Drop redundant check for downloaded file

The `fetch_file` commands are invoked for the `$($(package)_fetched)`
target, so the existence of the download stamp has already been tested.

hebasto committed:
depends: Use hash instead of file name for package download stamp

The package version is still included for convenience.

hebasto committed:
depends: Update URL for `qrencode` package source tarball

The https://fukuchi.org/ homepage no longer links to the source tarball,
and previously available files appear to have been removed. The homepage
now instructs users to download source tarballs from the GitHub releases
page instead.

The diff between the source trees is immaterial.

willcl-ark commented:
> > Can the `depends` CI cache be invalidated: (1) for this PR; (2) globally?
> 
> I would have thought the cache should have been invalidated automatically, looking into it.

So what has happened here is that the depends caches have missed on an exact cache hit, but restored the latest backup hit via:

https://github.com/bitcoin/bitcoin/blob/25212dfdb4cd7291392b6a94130f658c5bfa0a48/.github/actions/restore-caches/action.yml#L29-L31

It was my understanding that `make -C depends` was able to detect changes to the `*.mk` files and rebuild (changed) packages, but it seems that this is not the case when a cached .tar.gz is detected; `make` does not check if the recipe has changed.

Two possible fixes:

1. Update the makefile recipe to always calculate and use package definition hashes, even when a cached .tar.gz is detected
2. Neuter the CI cache restore logic to only restore exact cache matches (never fallback to "another hash"

IMO it would be more useful to attempt 1. as we could restore an old hit, and then only rebuild changed packages. This also feels useful for developers locally?

hebasto commented:
My Guix build:
```
aarch64
ff3ab842fd36d8c51e76fa87d80e460e3a26fca8641543f01be1b7b62eea99d8  guix-build-93a70a42d30f/output/aarch64-linux-gnu/SHA256SUMS.part
d8e022a8f8794df818cf682e75da7ca2f633e5954cd306cf14cd06f62bc3405d  guix-build-93a70a42d30f/output/aarch64-linux-gnu/bitcoin-93a70a42d30f-aarch64-linux-gnu-debug.tar.gz
c7ade0382153220a35b2d59a04f4c6578e40957898c468e0040555c49508039d  guix-build-93a70a42d30f/output/aarch64-linux-gnu/bitcoin-93a70a42d30f-aarch64-linux-gnu.tar.gz
9855573d20deb3346937bb56a8b494ed367c5bb56b53ca457b3997c06118c3b8  guix-build-93a70a42d30f/output/arm-linux-gnueabihf/SHA256SUMS.part
37645ca723b121a6e10d51d00be0d20bb39e1aeeac88dcd183eaf354b3b0bf44  guix-build-93a70a42d30f/output/arm-linux-gnueabihf/bitcoin-93a70a42d30f-arm-linux-gnueabihf-debug.tar.gz
aeb643212e09e5e0fa3e35732ecac097ad2cb9b0f47365ce813ffeac1a04a2dc  guix-build-93a70a42d30f/output/arm-linux-gnueabihf/bitcoin-93a70a42d30f-arm-linux-gnueabihf.tar.gz
a7d8ba23a581c99efc7879dfb1e9a297be11bfd3f5591ea76bb5df4d4579356e  guix-build-93a70a42d30f/output/arm64-apple-darwin/SHA256SUMS.part
ae1b89bd8c10386d925ce049b9fb0c7678f164cdeb083b3829e70986c15a08d6  guix-build-93a70a42d30f/output/arm64-apple-darwin/bitcoin-93a70a42d30f-arm64-apple-darwin-codesigning.tar.gz
dba540db96670e74a83d6ab3958b1bdb50722cba3c1062f462a40c031aa2dd88  guix-build-93a70a42d30f/output/arm64-apple-darwin/bitcoin-93a70a42d30f-arm64-apple-darwin-unsigned.tar.gz
9709ebf4cfa017b51a6f9bd7b1cba0b869b69a3d3c9ea3559a83f4b2409ed5ef  guix-build-93a70a42d30f/output/arm64-apple-darwin/bitcoin-93a70a42d30f-arm64-apple-darwin-unsigned.zip
dfdc1f25e2231702b591c9ae52b86023c942ccb660d059ed9db1b71f182e601e  guix-build-93a70a42d30f/output/dist-archive/bitcoin-93a70a42d30f.tar.gz
a7e38f598992df0b8bd3ebb1199a412c3537a4552f788d69a7b12eefce124386  guix-build-93a70a42d30f/output/powerpc64-linux-gnu/SHA256SUMS.part
a61961871a1a66c0a895c4d88db7f48904aefca5a1f0d8d824456bd44894c870  guix-build-93a70a42d30f/output/powerpc64-linux-gnu/bitcoin-93a70a42d30f-powerpc64-linux-gnu-debug.tar.gz
229212eeb5965c1e654c3750ad057644c7116b03d4d7632efa62d6947b58dc66  guix-build-93a70a42d30f/output/powerpc64-linux-gnu/bitcoin-93a70a42d30f-powerpc64-linux-gnu.tar.gz
a1df64b48c5e28d69f27341039d33a33998315a0a921b79556d29925f52a4ead  guix-build-93a70a42d30f/output/riscv64-linux-gnu/SHA256SUMS.part
65f8a7b4a04c2b179df92151b1ae3e96cd2b66ca0418a8718278647bafa11d91  guix-build-93a70a42d30f/output/riscv64-linux-gnu/bitcoin-93a70a42d30f-riscv64-linux-gnu-debug.tar.gz
971c05d110febfe8ecfebef066cf2f86bc40ec4e96cef317f04f847803d04082  guix-build-93a70a42d30f/output/riscv64-linux-gnu/bitcoin-93a70a42d30f-riscv64-linux-gnu.tar.gz
bfa7418c67a02ed744d22db867b15231369ed471b1a2b2b1d442491d352f68a2  guix-build-93a70a42d30f/output/x86_64-apple-darwin/SHA256SUMS.part
1471fcdaedc0ca978e22e48111fd8f3688d0ebb13ca681e0b4ee48334ff66262  guix-build-93a70a42d30f/output/x86_64-apple-darwin/bitcoin-93a70a42d30f-x86_64-apple-darwin-codesigning.tar.gz
aeb3f3bb1ac7b25a24445db7230892842c0f697910dfc5c0ba07cdca18f00f7d  guix-build-93a70a42d30f/output/x86_64-apple-darwin/bitcoin-93a70a42d30f-x86_64-apple-darwin-unsigned.tar.gz
41621033e0249b25cf976189d9dee2c790b1c30a1f6e372887c41fbcc04a44da  guix-build-93a70a42d30f/output/x86_64-apple-darwin/bitcoin-93a70a42d30f-x86_64-apple-darwin-unsigned.zip
4b1399bfddfaa8db0cdcd30edaeab0f8580071f4cf370f630825e52048b35cc7  guix-build-93a70a42d30f/output/x86_64-linux-gnu/SHA256SUMS.part
a687a433494cd31f1b1bec0427570154debf9664484e72d37c0ac39ef068af36  guix-build-93a70a42d30f/output/x86_64-linux-gnu/bitcoin-93a70a42d30f-x86_64-linux-gnu-debug.tar.gz
5baf6fa9cdd41cc4e07445b615085c4d7a86522be35e80efa5fa4a49557fe90a  guix-build-93a70a42d30f/output/x86_64-linux-gnu/bitcoin-93a70a42d30f-x86_64-linux-gnu.tar.gz
3308bda82c0064fb8a39819cf9e81008d22825936c0f5e39a483c166e76bded7  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/SHA256SUMS.part
13313b0acd9c597bef8e9eefeeaeb99e90345f1b2e6a6a88dd8c5c61c4592c78  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-codesigning.tar.gz
cb89ce2e732ac1b67c999176a599b3ab07e63bd3e878cfdf39d791cadd08c307  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-debug.zip
77dff472f4e87eb31b82857315360c92692350235707587fcaf8ecbed56a0e6d  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-setup-unsigned.exe
3fc3e9ca55bc656418f8b81c128c2c4f10c01ae9ee4b0fda619a6f1e3fe42f41  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-unsigned.zip
```

vasild reviewed (approved):
ACK 93a70a42d30fa2f9404b76d5bbdb5ea316fc1032

janb84 reviewed (commented):
ACK 93a70a42d30fa2f9404b76d5bbdb5ea316fc1032

PR Updates URL where the package can be downloaded for `qrencode` for depends. Because the package slightly differs from the original a new Hash has been added plus some changes to make sure the CI builds do not use the cache version. 

- Code review on the "added" shell scripts in the package to make sure that there has not been added some obvious malicious stuff. (yes package is from gh is from 2020 but still). ✅
- code review ✅

<details><summary>my Guix build</summary> 

**Host architecture:** `aarch64`
**Commit:** `93a70a42d30f`

```shell
 ff3ab842fd36d8c51e76fa87d80e460e3a26fca8641543f01be1b7b62eea99d8  guix-build-93a70a42d30f/output/aarch64-linux-gnu/SHA256SUMS.part
 d8e022a8f8794df818cf682e75da7ca2f633e5954cd306cf14cd06f62bc3405d  guix-build-93a70a42d30f/output/aarch64-linux-gnu/bitcoin-93a70a42d30f-aarch64-linux-gnu-debug.tar.gz
 c7ade0382153220a35b2d59a04f4c6578e40957898c468e0040555c49508039d  guix-build-93a70a42d30f/output/aarch64-linux-gnu/bitcoin-93a70a42d30f-aarch64-linux-gnu.tar.gz
 9855573d20deb3346937bb56a8b494ed367c5bb56b53ca457b3997c06118c3b8  guix-build-93a70a42d30f/output/arm-linux-gnueabihf/SHA256SUMS.part
 37645ca723b121a6e10d51d00be0d20bb39e1aeeac88dcd183eaf354b3b0bf44  guix-build-93a70a42d30f/output/arm-linux-gnueabihf/bitcoin-93a70a42d30f-arm-linux-gnueabihf-debug.tar.gz
 aeb643212e09e5e0fa3e35732ecac097ad2cb9b0f47365ce813ffeac1a04a2dc  guix-build-93a70a42d30f/output/arm-linux-gnueabihf/bitcoin-93a70a42d30f-arm-linux-gnueabihf.tar.gz
 a7d8ba23a581c99efc7879dfb1e9a297be11bfd3f5591ea76bb5df4d4579356e  guix-build-93a70a42d30f/output/arm64-apple-darwin/SHA256SUMS.part
 ae1b89bd8c10386d925ce049b9fb0c7678f164cdeb083b3829e70986c15a08d6  guix-build-93a70a42d30f/output/arm64-apple-darwin/bitcoin-93a70a42d30f-arm64-apple-darwin-codesigning.tar.gz
 dba540db96670e74a83d6ab3958b1bdb50722cba3c1062f462a40c031aa2dd88  guix-build-93a70a42d30f/output/arm64-apple-darwin/bitcoin-93a70a42d30f-arm64-apple-darwin-unsigned.tar.gz
 9709ebf4cfa017b51a6f9bd7b1cba0b869b69a3d3c9ea3559a83f4b2409ed5ef  guix-build-93a70a42d30f/output/arm64-apple-darwin/bitcoin-93a70a42d30f-arm64-apple-darwin-unsigned.zip
 dfdc1f25e2231702b591c9ae52b86023c942ccb660d059ed9db1b71f182e601e  guix-build-93a70a42d30f/output/dist-archive/bitcoin-93a70a42d30f.tar.gz
 a7e38f598992df0b8bd3ebb1199a412c3537a4552f788d69a7b12eefce124386  guix-build-93a70a42d30f/output/powerpc64-linux-gnu/SHA256SUMS.part
 a61961871a1a66c0a895c4d88db7f48904aefca5a1f0d8d824456bd44894c870  guix-build-93a70a42d30f/output/powerpc64-linux-gnu/bitcoin-93a70a42d30f-powerpc64-linux-gnu-debug.tar.gz
 229212eeb5965c1e654c3750ad057644c7116b03d4d7632efa62d6947b58dc66  guix-build-93a70a42d30f/output/powerpc64-linux-gnu/bitcoin-93a70a42d30f-powerpc64-linux-gnu.tar.gz
 a1df64b48c5e28d69f27341039d33a33998315a0a921b79556d29925f52a4ead  guix-build-93a70a42d30f/output/riscv64-linux-gnu/SHA256SUMS.part
 65f8a7b4a04c2b179df92151b1ae3e96cd2b66ca0418a8718278647bafa11d91  guix-build-93a70a42d30f/output/riscv64-linux-gnu/bitcoin-93a70a42d30f-riscv64-linux-gnu-debug.tar.gz
 971c05d110febfe8ecfebef066cf2f86bc40ec4e96cef317f04f847803d04082  guix-build-93a70a42d30f/output/riscv64-linux-gnu/bitcoin-93a70a42d30f-riscv64-linux-gnu.tar.gz
 bfa7418c67a02ed744d22db867b15231369ed471b1a2b2b1d442491d352f68a2  guix-build-93a70a42d30f/output/x86_64-apple-darwin/SHA256SUMS.part
 1471fcdaedc0ca978e22e48111fd8f3688d0ebb13ca681e0b4ee48334ff66262  guix-build-93a70a42d30f/output/x86_64-apple-darwin/bitcoin-93a70a42d30f-x86_64-apple-darwin-codesigning.tar.gz
 aeb3f3bb1ac7b25a24445db7230892842c0f697910dfc5c0ba07cdca18f00f7d  guix-build-93a70a42d30f/output/x86_64-apple-darwin/bitcoin-93a70a42d30f-x86_64-apple-darwin-unsigned.tar.gz
 41621033e0249b25cf976189d9dee2c790b1c30a1f6e372887c41fbcc04a44da  guix-build-93a70a42d30f/output/x86_64-apple-darwin/bitcoin-93a70a42d30f-x86_64-apple-darwin-unsigned.zip
 4b1399bfddfaa8db0cdcd30edaeab0f8580071f4cf370f630825e52048b35cc7  guix-build-93a70a42d30f/output/x86_64-linux-gnu/SHA256SUMS.part
 a687a433494cd31f1b1bec0427570154debf9664484e72d37c0ac39ef068af36  guix-build-93a70a42d30f/output/x86_64-linux-gnu/bitcoin-93a70a42d30f-x86_64-linux-gnu-debug.tar.gz
 5baf6fa9cdd41cc4e07445b615085c4d7a86522be35e80efa5fa4a49557fe90a  guix-build-93a70a42d30f/output/x86_64-linux-gnu/bitcoin-93a70a42d30f-x86_64-linux-gnu.tar.gz
 3308bda82c0064fb8a39819cf9e81008d22825936c0f5e39a483c166e76bded7  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/SHA256SUMS.part
 13313b0acd9c597bef8e9eefeeaeb99e90345f1b2e6a6a88dd8c5c61c4592c78  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-codesigning.tar.gz
 cb89ce2e732ac1b67c999176a599b3ab07e63bd3e878cfdf39d791cadd08c307  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-debug.zip
 77dff472f4e87eb31b82857315360c92692350235707587fcaf8ecbed56a0e6d  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-setup-unsigned.exe
 3fc3e9ca55bc656418f8b81c128c2c4f10c01ae9ee4b0fda619a6f1e3fe42f41  guix-build-93a70a42d30f/output/x86_64-w64-mingw32/bitcoin-93a70a42d30f-win64-unsigned.zip
```
</detais>
